src/model/model.h

```diff
@@ -8,7 +8,7 @@ struct Point {
     double y;
     Point(double x, double y);
 
-    bool operator == (const Point & p);
```

Comment:
what is this doing here?

Reply:
if you mean the `const` keyword, it makes sure the compare operator doesn't change anything - which obviously shouldn't happen in the first place